commerce_coordinator/apps/lms/serializers.py

```diff
@@ -0,0 +1,17 @@
+"""Serializers for LMS (edx-platform) service"""
+from rest_framework import serializers
+
+
+# Originally stolen verbatim from Ecomm
+class OrderCreatedSignalInputSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer for order_created_signal input validation.
+    """
+    product_sku = serializers.ListField(
+        child=serializers.CharField(), allow_empty=False
+    )
+    edx_lms_user_id = serializers.IntegerField(allow_null=False)
+    email = serializers.EmailField(allow_null=False)
+    first_name = serializers.CharField(allow_null=False)
+    last_name = serializers.CharField(allow_null=False)
```

Comment:
These 2 fields are nullable in the User model (in LMS as well as in Coordinator). So we not make them mandatory.

We just need to confirm from Titans that their cart creation API will not through an error for not providing these fields.

Reply:
ive removed the allow null prohibition for now while i look in to the down line on this.

Reply:
We may need to fix this forward if they say no (as in currently).

Reply:
Can we catchup tomorrow on exactly what you mean? (incase im understanding wrongly)

Reply:
Question asked, https://twou.slack.com/archives/C03S3D8736H/p1684430310470109

(sorry pasted a message i had prepped for another field, so you get 2 emails)